src/database/data-source.ts

```
@@ -0,0 +1,20 @@
+import { SnakeNamingStrategy } from 'typeorm-naming-strategies';
+
+dotenv.config();
+
+export const AppDataSource = new DataSource({
+  type: 'postgres',
+  host: process.env.DB_HOST || 'localhost',
+  port: Number(process.env.DB_PORT) || 5432,
+  username: process.env.DB_USERNAME || 'postgres',
+  password: process.env.DB_PASSWORD || '',
+  database: process.env.DB_NAME || 'myapp',
+  entities: [__dirname + '/../entities/*.entity.{ts,js}'],
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)